tests/ui_tools/test_boxes.py

```diff
@@ -68,6 +68,53 @@ def test_not_calling_send_private_message_without_recipients(self, key,
 
         assert not write_box.model.send_private_message.called
 
+    @pytest.mark.parametrize('key', keys_for_command('SAVE_AS_DRAFT'))
+    @pytest.mark.parametrize('draft_composition', [
+        {
+            'type': 'stream',
+            'to': 'Current Stream',
+            'content': 'Random message',
+            'subject': 'Topic'
+        },
+        {
+            'type': 'private',
+            'to': ['emailgateway@zulip.com'],
+            'content': 'Random message'
+        },
```

Comment:
Maybe add one test fixture for PM huddle too?

Reply:
We have only two data structures `PrivateComposition` and `StreamComposition`. Testing already covers both these cases. For PM huddle, its just a case of appending more emails to the existing list. Is there a need for a separate parameter?